dts/bindings/regulator/regulator-fixed.yaml

```diff
@@ -0,0 +1,49 @@
+# Copyright 2019-2020 Peter Bigot Consulting, LLC
+# SPDX-License-Identifier: Apache-2.0
+
+description: GPIO-controlled regulators
+
+include: regulator.yaml
+
+# NOTE: The driver supports "regulator-fixed-sync" as a specializing
+# variant when it's known that the GPIO state change operations are
+# isr-ok and not sleep, and both startup and off-on delays are zero.
+# This bypasses the asynchronous onoff manager which optimizes both time
+# and memory use.
+#
+# To enable this use both:
+#    compatible = "regulator-fixed-sync", "regulator-fixed";
+#
+
+compatible: "regulator-fixed"
```

Comment:
Are we trying to be in sync with the linux binding for `regulator-fixed`?

Reply:
To a reasonable degree, yes.  We don't support external properties for the active level and open drain, for reasons described in the binding.